sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -9,110 +9,36 @@ import {
-        border-width: 0 0 var(--border-width);
-        padding: 0 calc(var(--design-unit) * 2px);
-    }
-
-    .framework_ContentPlacement:last-of-type {
-        border-bottom: none;
-    }
-
-    @media screen and (min-width: 700px) {
-        .framework_ContentPlacement {
-            border-width: 0 var(--border-width) var(--border-width) 0;
-        }
+    :host([section="framework"]) {
+        grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+        contain: content;
+        overflow: hidden;
     }
 
-    @media screen and (max-width: 900px) {
-        :host {
-            --flow: row;
-        }
-    }
-
-    @media screen and (min-width: 700px) and (max-width: 999px) {
-        .framework_container {
-            grid-template-columns: repeat(2, 1fr);
-        }
-
-        .community_container {
-            grid-template-columns: repeat(3, 1fr);
-        }
-
-        .framework_ContentPlacement:nth-of-type(even) {
-            border-right: none;
-        }
-
-        .framework_ContentPlacement:nth-last-of-type(-n + 2):nth-of-type(odd) {
-            /* The last two items on medium screens have no bottom border */
-            border-bottom: none;
-        }
-    }
-
-    @media screen and (min-width: 1000px) {
-        .framework_container,
-        .community_container {
-            grid-template-columns: repeat(3, 1fr);
-        }
-
-        .framework_ContentPlacement:nth-of-type(3n),
-        .framework_ContentPlacement:first-of-type
-            + .framework_ContentPlacement:last-of-type,
-        .framework_ContentPlacement:only-of-type {
-            border-right: none;
-        }
-
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n):nth-last-of-type(-n + 3), /* first item when only three items are present */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n) ~ .framework_ContentPlacement:nth-last-of-type(-n + 3), /* last three items when full row of three */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n - 1) ~ .framework_ContentPlacement:nth-last-of-type(-n + 2), /* more than one row, last row with only two items */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n - 2) ~ .framework_ContentPlacement:last-of-type, /* more than one row, last row with only one item */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(-n + 2) {
-            /* only two items on a row that should fit three */
-            border-bottom: none;
-        }
-
-        .framework_ContentPlacement:nth-of-type(6) {
-            border: none;
-        }
+    :host([section="community"]) {
+        grid-template-columns: repeat(auto-fit, minmax(225px, 1fr));
     }
 
     /* This creates the color, background, and elevation changes on hover */
-    /* start */
-    .community_container:hover site-content-placement,
-    .feature_container:hover site-feature-card {
+
+    :host([section="community"]:hover) site-content-placement {
```

Comment:
I believe that we should  still include feature on hover here. on hover all not hovered elements should be greyed out and currently why are being displayed as such. 
![Screen Shot 2020-06-02 at 12 55 55 PM](https://user-images.githubusercontent.com/48499261/83565311-0f4d4b80-a4d3-11ea-8ff9-5a1790ba35cc.png)

Reply:
Are those links actionable while greyed out? If so, we need to pass WCAG 2.1 AA accessibility requirements for contrast. If they are *not* actionable, how do I interact with them as a keyboard user when hover is not available to me? If they are not actionable, can we not show them?

Reply:
They are actionable in the sense if you move over to them then they are hovered upon and you can click them. But you can not click on the links without invoking hover on the card and making the links active. If there was a user that could not use hover, all of the links are actionable because the greying out of the elements only occurs on hover. I have added an image of the figma to explain what I am talking about. However, I do think that it would be a bit more clear/more accessible if in the design we had the links for the at rest card in an accent color.
<img width="608" alt="Screen Shot 2020-06-02 at 2 05 54 PM" src="https://user-images.githubusercontent.com/48499261/83570271-98b44c00-a4da-11ea-99f0-0765aadfd5db.png">

Reply:
The links are actionable.  In the original design, they were not visible by default and visible on hover.  With the updated design I made them visible with the accent color by default.  When the section is hovered, the links of all but the currently hovered card desaturate.  That was done in a prior PR, not this one.  If there are changes to be made, would it be preferred to incorporate them into this PR or do a separate one addressing that change specifically?

Reply:
currently in feature the content it greyed out on hover. see below. This PR changes the color.
![Screen Shot 2020-06-02 at 3 51 03 PM](https://user-images.githubusercontent.com/48499261/83577437-023b5700-a4e9-11ea-83a9-1467e296a603.png)

Reply:
neutral-foreground-hint returned to feature section hover state